azure/v11.4.0/release.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: v11.4.0
+spec:
+  state: active
+  date: 2020-06-23T15:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: chart-operator
+    version: 0.13.0
+  - name: coredns
+    componentVersion: 1.6.5
+    version: 1.1.8
+  - name: external-dns
+    componentVersion: 0.5.18
+    version: 1.2.1
+  - name: kube-state-metrics
+    componentVersion: 1.9.5
+    version: 1.1.0
+  - name: metrics-server
+    componentVersion: 0.3.3
+    version: 1.1.0
+  - name: net-exporter
+    version: 1.8.1
+  - name: nginx-ingress-controller
+    componentVersion: 0.30.0
+    version: 1.6.12
+  - name: node-exporter
+    componentVersion: 0.18.1
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: azure-operator
+    version: 4.1.0-dev
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 0.23.10-dev
```

```suggestion
    version: 0.23.10
```

TODO apply once operator is released; until then, deploy the branch for testing